vms/rpcchainvm/vm_client.go

```diff
@@ -392,6 +392,25 @@ func (vm *VMClient) CreateHandlers(ctx context.Context) (map[string]http.Handler
 	return handlers, nil
 }
 
+func (vm *VMClient) CreateGRPCService(ctx context.Context) (string, http.Handler, error) {
+	resp, err := vm.client.CreateGRPCService(ctx, &emptypb.Empty{})
+	if err != nil {
+		return "", nil, err
+	}
+
+	if resp.ServiceName == "" && resp.ServerAddr == "" {
```

Comment:
We're doing this instead of comparing against the zero-value of the response because proto responses auto-embed `protoimpl.MessageState` which cannot be compared.

If one of these was empty and the other wasn't, it would be because the vm explicitly tried to register a grpc service with an empty service name which I think should be invalid (since grpc calls take the form of service + method).